### DISCUSSION TITLE
ZKSync's EIP-1559 Support

### DISCUSSION THREAD

**Discussion by FelixFan1992** [zkSync-Community-Hub/zksync-developers#610] [General] [answered]
### Environment

Testnet

### Provide a brief description of the functionality you're trying to implement and the issue you are running into.

In this [document](https://docs.zksync.io/zk-stack/concepts/transaction-lifecycle), it means that 
`In ZKsync Era, while the EIP-1559 transaction format is supported, the maxFeePerGas and maxPriorityFeePerGas parameters are not utilized.`
Can you explain more about this? Does it literally mean that only the base fee is supported? If so, any plans to start using them?
If a tx is sent with only legacy gas price value, will zksync treat it as a base fee value?

**Comment by FelixFan1992** [comment]
The direct reason I ask is I ran into this error when using `era_test_node` on my failed tx: `Error description: Block.basefee is greater than max fee per gas`

**Reply by bxpana** [reply]
> Does it literally mean that only the base fee is supported?

That is my current understanding, but will check on that.

**Reply by bxpana** [reply, marked as answer]
Appreciate your patience, but was able to get an answer for you. 

> If a tx is sent with only legacy gas price value, will zksync treat it as a base fee value?

Right now, yes. We just charge the base fee, but in the future we will probably use the priority fee-related fields similar to Ethereum
